Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -1306,10 +1311,21 @@ Codeunit 4037 "Helper Functions"
                 codeunit.Run(codeunit::"Gen. Jnl.-Post Batch", GenJournalLine);
     end;
 
+    local procedure PostPurchBatch()
+    var
+        ItemJournalLine: Record "Item Journal Line";
+    begin
+        ItemJournalLine.SetRange("Journal Batch Name", PurchBatchNameTxt);
```

Comment:
don't need template name for item journal line?

Reply:
I don't think it's really needed in this case, it seemed unnecessary.